freshmaker/utils.py

```diff
@@ -189,3 +190,21 @@ def _run_command(command, logger=None, rundir=None, output=subprocess.PIPE, erro
 def is_pkg_modular(nvr):
     """ Returns True if the package is modular, False otherwise. """
     return "module+" in nvr
+
+
+def get_ocp_release_date(ocp_version):
+    """ Get product release data via Product Page API
+
+    :param str ocp_version: ocp version
+    :return: date in format of "%Y-%m-%d", example: 2021-02-23
+    :rtype: str
+    """
+    if not conf.product_page_api_url:
+        raise RuntimeError("Product Page API url is not set in config")
+
+    ocp_release = f"openshift-{ocp_version}"
+
+    url = f"{conf.product_page_api_url.rstrip('/')}/releases/{ocp_release}/schedule-tasks"
+    resp = requests.get(url, params={"name": "GA", "fields": "name,date_finish"})
```

Comment:
Could you please specify a [timeout](https://requests.readthedocs.io/en/master/user/quickstart/#timeouts)?